src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts

```diff
@@ -200,10 +206,6 @@ export class JJDisputeComponent implements OnInit {
       this.contactInformationForm.patchValue(this.lastUpdatedJJDispute);
       this.courtOptionsForm.patchValue(this.lastUpdatedJJDispute);
 
-      if (this.jjIDIR === this.lastUpdatedJJDispute.lockedBy) {
-        this.startTimer(this.lastUpdatedJJDispute.lockExpiresAtUtc, this.lastUpdatedJJDispute.lockId);

```

Comment:
Shouldn't we still use the lock on decision validation screen only if this is a shared view?

Reply:
This implementation has shifted to backend, we are not using this in front end